google/cloud/video/transcoder_v1/types/resources.py

```diff
@@ -74,8 +81,10 @@ class Job(proto.Message):
             -  ``preset/{preset_id}``
 
             -  User defined JobTemplate: ``{job_template_id}``
+            This field is a member of `oneof`_ ``job_config``.
```

Comment:
The kokoro build is failing with `/usr/local/google/home/partheniou/git/python-video-transcoder/google/cloud/video/transcoder_v1/types/resources.py:docstring of google.cloud.video.transcoder_v1.types.resources.Job:47:Bullet list ends without a blank line; unexpected unindent.`

The build passes if I add a line break above this line (and corresponding line in v1beta1).